### ISSUE TITLE
Bug: Server crashes on start

### ISSUE THREAD

**Issue by skamensky** [cortezaproject/corteza-server-corredor#45]
When starting up the server using tag version `2022.9.0-dev.4`. I made no other change besides for the version (upgraded from `2022.9.0-dev.2`), this was previously working.

The server immediately crashes with the following error:

```
/corredor/node_modules/chartjs-plugin-colorschemes/src/colorschemes/index.js:3
import * as brewer from './colorschemes.brewer';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at wrapSafe (internal/modules/cjs/loader.js:1001:16)
    at Module._compile (internal/modules/cjs/loader.js:1049:27)
    at Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Object.require.extensions.<computed> [as .js] (/corredor/node_modules/ts-node/src/index.ts:851:44)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at Object.<anonymous> (/corredor/node_modules/@cortezaproject/corteza-js/src/shared/types/chart/colorschemes.ts:2:22)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
corteza_corredor_1 exited with code 1
```

**Comment by skamensky** [comment]
@Fajfa 

Did you see this issue?

It's still occuring on `2022.9.0-rc.2`

**Comment by Fajfa** [comment]
Did not see it no, someone else is already working on a fix regarding this

**Comment by skamensky** [comment]
Great thank you

**Comment by Fajfa** [comment]
@skamensky 
Could you double check this, we can't seem to replicate it. 
Even if trying to import what you are importing (not sure where that comes from if you don't import it)

**Comment by skamensky** [comment]
I can reproduce it even by just running `docker run cortezaproject/corteza-server-corredor:2022.9.0-dev.4`
Do you not get that same error?

You can go to https://labs.play-with-docker.com/ for a clean tetsing environment

**Comment by Fajfa** [comment]
I didn't manage to, will try again. Thanks

**Comment by skamensky** [comment]
Now did you?

**Comment by Fajfa** [comment]
Someone else is working on this task. They have managed to replicate it and you can expect the fix to be in 2022.9.1
Which should be released sometime in the middle of the next week

**Comment by skamensky** [comment]
Great and thanks. No rush. I just wanted to make sure you could reproduce it.

**Comment by Fajfa** [comment]
This should be fixed in the `2022.9.1` version. Thank you for raising the isssue

**Comment by skamensky** [comment]
Confirming that 2022.9.1 does fix this issue
